### PR TITLE
NETOBSERV-1965: pkt translation enrichment

### PR DESCRIPTION
**Pull request by msherif1234**
## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [ ] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [ ] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

File: web/src/utils/columns.ts [hunk 1]
```diff
@@ -44,7 +46,7 @@ export enum ColumnsId {
   dscp = 'Dscp',
   tcpflags = 'TCPFlags',
   bytes = 'Bytes',
-  packets = 'Packets',
+  packets = 'Padstkubeobjectckets',
```
Comment (jpinsonneau) [inline web/src/utils/columns.ts:49]:
```suggestion
  packets = 'Packets',
```

**Review by OlivierCazade — APPROVED** [review]

File: web/src/api/ipfix.ts [hunk 2]
```diff
@@ -179,6 +179,14 @@ export interface Fields {
...
   /** In conversation tracking, a counter of flow logs per conversation */
   numFlowLogs?: number;
+  /** Xlat Source IP address (ipv4 or ipv6) */
+  XlatSrcAddr?: string;
+  /** Xlat Destination IP address (ipv4 or ipv6) */
+  XlatDstAddr?: string;
+  /** Xlat Source port */
+  XlatSrcPort?: number;
+  /** Xlat Destination port */
+  XlatDstPort?: number;
```
Comment (jotak) [inline web/src/api/ipfix.ts:189]: you can do that but it's not required anymore since it's defined from config

File: web/src/utils/columns.ts [hunk 3]
```diff
@@ -27,6 +27,8 @@ export enum ColumnsId {
   kubeobject = 'K8S_Object',
   srckubeobject = 'SrcK8S_Object',
   dstkubeobject = 'DstK8S_Object',
+  xlatsrckubeobject = 'XlatSrcK8S_Object',
+  xlatdstkubeobject = 'XlatDstK8S_Object',
```
Comment (jotak) [inline web/src/utils/columns.ts:31]: same here I think it's not needed, so I'd say the whole PR is not needed

**Comment by msherif1234** [comment]
not needed anymore will close it

**Pull request closed without merging**